base/src/main/java/com/smartdevicelink/proxy/rpc/WindowState.java

```diff
@@ -0,0 +1,49 @@
+package com.smartdevicelink.proxy.rpc;
+
+import android.support.annotation.NonNull;
+
+import com.smartdevicelink.proxy.RPCStruct;
+
+import java.util.Hashtable;
+
+public class WindowState extends RPCStruct {
+    public static final String KEY_APPROXIMATE_POSITION = "approximatePosition";
+    public static final String KEY_DEVIATION = "deviation";
+
+    /**
+     * Constructs a newly allocated WindowStatus object
+     */
+    public WindowState() {}
+
+    /**
+     * Constructs a newly allocated WindowStatus object indicated by the Hashtable parameter
+     * @param hash The Hashtable to use
+     */
+    public WindowState(Hashtable<String, Object> hash) {
+        super(hash);
+    }
+    /**
+     * Constructs a newly allocated WindowStatus object
+     */
+    public WindowState(@NonNull Integer approximatePosition, @NonNull Integer deviation) {
+        this();
+        setApproximatePosition(approximatePosition);
+        setDeviation(deviation);
+    }
+
+    public void setApproximatePosition(Integer approximatePosition) {
+        setValue(KEY_APPROXIMATE_POSITION, approximatePosition);
+    }
+
```

Comment:
```suggestion
    /**
     * Gets the approximatePosition.
     *
     * @return Integer The approximate percentage that the window is open - 0 being fully closed, 100 being fully
     * open
     */
```
Please include Javadocs